src/api/guardian.js

```diff
@@ -0,0 +1,17 @@
+const guardian = require("guardian-js");
+require("env2")("./config.env");
```

Comment:
i would move this into the server start script for you only need to do it once

Reply:
I though you needed to use this whenever you are requesting  env variables

Reply:
Didn't work!

Reply:
Worked!

Reply:
you only need to do it once since it will only register the environment variable on the server once, so just move it to the start script